plugins/inputs/intel_dlb/README.md

```diff
@@ -0,0 +1,118 @@
+# Intel® Dynamic Load Balancer (Intel® DLB)  Input Plugin
+
+The `Intel DLB` plugin collects metrics exposed by applications built with
+[Data Plane Development Kit](https://www.dpdk.org/) which is an extensive
+set of open source libraries designed for accelerating packet processing
+workloads, plugin is also using bifurcated driver. More specifically it's
+targeted for applications that use Intel DLB as eventdev devices accessed
+via bifurcated driver (allowing access from kernel and user-space).
+
+## Metrics
+
+There are two sources of metrics:
+
+- DPDK-based app for detailed eventdev metrics per device, per port and per queue
+- Sysfs entries from kernel driver for RAS metrics
+
+## About Intel® Dynamic Load Balancer (Intel® DLB)
+
+The Intel® Dynamic Load Balancer (Intel® DLB) is a PCIe device that provides
+load-balanced, prioritized scheduling of events (that is, packets) across
+CPU cores enabling efficient core-to-core communication. It is a hardware
+accelerator located inside the latest Intel® Xeon® devices offered by Intel.
+It supports the event-driven programming model of DPDK's Event Device Library.
+This library is used in packet processing pipelines for multi-core scalability,
+dynamic load-balancing, and variety of packet distribution and synchronization
+schemes.
+
+## About DPDK Event Device Library
+
+The DPDK Event device library is an abstraction that provides the application
+with features to schedule events. The eventdev framework introduces the event
+driven programming model. In a polling model, lcores poll ethdev ports and
+associated Rx queues directly to look for a packet. By contrast in an event
+driven model, lcores call the scheduler that selects packets for them based on
+programmer-specified criteria. The Eventdev library adds support for an event
+driven programming model, which offers applications automatic multicore scaling,
+dynamic load balancing, pipelining, packet ingress order maintenance and
+synchronization services to simplify application packet processing.
+By introducing an event driven programming model, DPDK can support
+both polling and event driven programming models for packet processing,
+and applications are free to choose whatever model (or combination of the two)
+best suits their needs.
+
+## Pre-configuration
```

Comment:
Is any user able to read these sysfs entires and read the dpdk socket? Or do we need a note in here about the run-time user/group?

Reply:
Proper note added to `README.md`.